controllers/datadogagent/feature/apm/feature_test.go

```diff
@@ -23,8 +23,8 @@ import (
 )
 
 const (
-	apmSocketHostPath = "/var/run/datadog/apm"
-	apmSocketPath     = "/var/run/datadog/apm/apm.sock"
+	apmSocketHostPath = "/var/run/datadog"
+	apmSocketPath     = "/var/run/datadog/apm.sock"
```

Comment:
These are not technically necessary - You should rename `DogstatsdSocketVolumePath` in `DogstatsdAPMSocketVolumePath` in apis/datadoghq/common/const.go and you have `apmSocketPath` defined using `DogstatsdAPMSocketVolumePath` and `APMSocketName` too.
This is a test file so it's not the most important.